src/extensions/surveys.tsx

```diff
@@ -280,6 +283,7 @@ const questionTypeMap = (
         [SurveyQuestionType.MultipleChoice]: (
             <MultipleChoiceQuestion
                 question={question as MultipleSurveyQuestion}
+                choices={getDisplayOrderChoices(question as MultipleSurveyQuestion)}
```

Comment:
not a big fan of this interface, since this seems like an internal detail of `MultipleChoiceQuestion`, given we're using the question input, and only the question input to determine what choices to display.

This is prone to bugs if I say forget to pass in choices, or pass in the wrong ones. 

I'd suggest making this an internal detail, and doing the choice ordering inside this component.